servers/src/main/webapp/memory.jsp

```diff
@@ -34,6 +35,7 @@
                   <th><%= fileInfo.getAbsolutePath() %></th>
                   <th><%= fileInfo.getSize() %></th>
                   <th><%= fileInfo.getBlockSizeBytes() %></th>
+                  <th><%= fileInfo.getUsername() %></th>
```

Comment:
Yes, we should display group too. Since group mapping is still not added, shall we leave group name in follow-up PRs?  Otherwise, group is empty now to display.